### PR TITLE
Add RDS Cluster and DB snapshots and attributes

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

**Review by roneli — CHANGES_REQUESTED** [review]
Looks good, 1 change required, and e2e tests.

File: resources/rds_db_snapshots.go [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+			{
+				Name:        "attributes",
+				Description: "Snapshot attribute names and values",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveRDSDBSnapshotAttributes,
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "aws_rds_db_snapshot_processor_features",
```
[the remark this answers is not in the document]
Comment (irmatov) [reply resources/rds_db_snapshots.go]: converted to json field

**Review by roneli — APPROVED** [review]
LGTM, add e2e tests and we can merge

**Comment by irmatov** [comment]
e2e tests added

**Pull request merged**